src/CliCore.ts

```diff
@@ -31,7 +31,7 @@ export default class CliCore {
     constructor(templateProcessor: TemplateProcessor = null) {
         this.templateProcessor = templateProcessor;
         this.logLevel = "info";
-        this.currentDirectory = path.join(process.cwd(), 'example'); // Default to cwd/example
+        this.currentDirectory = process.cwd();
```

Comment:
I think defaulting to example directory was quite good experience for most cases.